### DISCUSSION TITLE
Missing an imagePullSecret parameter for Helm Chart

### DISCUSSION THREAD

**Discussion by lepouletsuisse** [jertel/elastalert2#245] [Q&A] [answered]
Hey guys,
In my current company, we are using our own private Docker Container Registry for security reasons. We are also using Helm chart for our deployments but it seems that the current Elastalert2 Helm Chart doesn't allow us to specify an `imagePullSecret` parameter, allowing us to use our private registry. 
Is there anything planned for this feature?

I can do a PR if needed.

**Comment by jertel** [comment, marked as answer]
You can set image pull secrets at the service account level: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account

If you still prefer to set them at each deployment then yes, please submit a PR.

**Reply by lepouletsuisse** [reply]
This is all good, I didn't know setting the imagePullSecret on the ServiceAccount was possible. Thank you for the trick, this is working fine :+1:
